docs/source/dataframe-aggregations.rst

```diff
@@ -0,0 +1,83 @@
+Dataframe Aggregations
+======================
+
+Dataframe aggregations are composed of an aggregation (like sum, mean, ...) and
+a windowing scheme (fixed sized windows, index-valued, all time, ...)
+
+Aggregations
+------------
+
+Streaming Dataframe aggregations are built from three methods
+
+-  ``initial``: Creates initial state given an empty example dataframe
+-  ``on_new``: Updates state and produces new result to emit given new data
```

Comment:
It can be any type.  If it is a series or dataframe you'll get back the appropriate streaming type.  Otherwise you'll get the base `Streaming` type found in `collections.py`